pkg/pipelines/oc_adm.go

```diff
@@ -0,0 +1,74 @@
+package pipelines
+
+import (
+	"fmt"
+
+	securityv1typedclient "github.com/openshift/client-go/security/clientset/versioned/typed/security/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apiserver/pkg/authentication/serviceaccount"
+)
+
+// interface to invoke methods on SCC
+type sccAccessor interface {
+	addSCCToUser(sccName, namespace, saName string) error
+}
+
+type scc struct {
+	client   securityv1typedclient.SecurityContextConstraintsInterface
+	accessor sccAccessor
+}
+
+// newSCC creates a sccOperations
+func newSCC() (*scc, error) {
+
+	// obtain client config
+	clientConfig, err := getClientConfig()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get client config due to %w", err)
+	}
+
+	// obtain security client
+	securityClient, err := securityv1typedclient.NewForConfig(clientConfig)
+	if err != nil {
+		return nil, fmt.Errorf("failed to obtain Security Client due to %w", err)
```

Comment:
The Go standard is simpler than this:

```suggestion
		return nil, fmt.Errorf("failed to obtain Security Client: %w", err)
```

Reply:
I just searched "due to" in the same project, there is *lots* of matches in the vendored source.   The term "due to" is very common used in logging error messages in go code or any languages for that matter.  There is nothing wrong with "due to".   I don't have problem replacing it (I alreadydid) but you have to understand how commonly accepted is to use "due to" to log the cause of error.